.github/workflows/ci.yml

```diff
@@ -28,6 +28,8 @@ on:
 
 env:
   REPORTS_DIR: CI_reports
+  PYTHON_VERISON_DEFAULT: "3.10"
```

Comment:
@madpah you might want to take this aproach over to the py-lib
using env vars so the several versions could be maintained easily in the github-action

Reply:
👍